tests/1_challangeExit.js

```diff
@@ -0,0 +1,98 @@
+  let aliceUnspent = unspentForAddress(state.unspent, alice, color).map(u => ({
+    output: u.output,
+    outpoint: Outpoint.fromRaw(u.outpoint),
+  }));
+  let inputs = [new Input(aliceUnspent[0].outpoint)];
+  let outputs = [new Output(amount, alice.toLowerCase(), color)];
+  let firstTransfer = Tx.transfer(inputs, outputs).signAll(alicePriv);
+
+  await node.sendTx(firstTransfer.hex());
+  await sleep(3000);
+
+  state = await node.getState();
+  console.log(state);
+  aliceUnspent = unspentForAddress(state.unspent, alice, color).map(u => ({
+    output: u.output,
+    outpoint: Outpoint.fromRaw(u.outpoint),
+  }));
+  inputs = [new Input(aliceUnspent[0].outpoint)];
+  outputs = [new Output(amount, alice.toLowerCase(), color)];
+  let secondTransfer = Tx.transfer(inputs, outputs).signAll(alicePriv);
+
+  await node.sendTx(secondTransfer.hex());
+  await sleep(3000);
+
+  state = await node.getState();
+  console.log(state);
+
+  console.log("Make some more deposits to make sure the block is submitted (with log is off)...")
+  for (let i = 0; i < 20; i++) {
+    await mintAndDeposit(zzz, i + 1, minter, contracts.token, contracts.exitHandler, true);
+    await sleep(1000);
+  }
+  await sleep(3000);
+
+  const latestBlockNumber = (await node.web3.eth.getBlock('latest')).number;
+  console.log("Latest Block number: ", latestBlockNumber);
+  const latestSubmittedBlock = latestBlockNumber - latestBlockNumber % 32;
+  console.log("Latest submitted block number: ", latestSubmittedBlock);
+  if (latestSubmittedBlock === 0) {
+    throw new Error("Can't exit, no periods were submitted yet");
+  };
+
+  //exit
+  let txData = await node.web3.eth.getTransaction(firstTransfer.hash());
+  const validatorInfo = await node.web3.getValidatorInfo();
+  const proof = await helpers.getProof(node.web3, txData, 0, validatorInfo.ethAddress);
+  const youngestInputTx = await helpers.getYoungestInputTx(node.web3, Tx.fromRaw(txData.raw));
+  const inputProof = await helpers.getProof(node.web3, youngestInputTx.tx, 0, validatorInfo.ethAddress);
+
+  const balanceBefore = await contracts.token.methods.balanceOf(alice).call();
+  const plasmaBalanceBefore = await node.web3.eth.getBalance(alice);
+  console.log("Account mainnet balance: ", balanceBefore);
+  console.log("Account plasma balance: ", plasmaBalanceBefore);
+  console.log("Attempting exit...");
+  const exitTx = await contracts.exitHandler.methods.startExit(
+      inputProof,
+      proof,
+      0,
+      youngestInputTx.index
+  ).send({from: alice, value: 100000000000000000, gas: 2000000});
+  const utxoId = new Outpoint(firstTransfer.hash(), 0).getUtxoId();
+  let exit = await contracts.exitHandler.methods.exits(utxoId).call();
+  console.log(exit);
+
+  await challenge(firstTransfer.hash(), 
+    secondTransfer.hash(), 
+    node.web3.currentProvider.host, 
+    web3.currentProvider.host, 
+    challengerPriv);
+
+  exit = await contracts.exitHandler.methods.exits(utxoId).call();
+  console.log(exit);
```

Comment:
should we check here that `challenge` removed the fraudlent exit from the queue?

Reply:
Yeah was about to do that.